### PR TITLE
Update README and PR template

### PR DESCRIPTION
**Pull request by arsalansufi**
This PR updates the README and PR template given other recent updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,79 +4,84 @@
...
-## Developing
+## Development
 
-To run the demo of the boilerplate code here, follow these steps:
-
-Pull the repository: `git clone git@github.com:Clever/{{.AppName}}.git`
+```
+cd {{.AppName}}
+npm install
+ark start family-portal -e clever-dev -l
```
Comment (adamvictorclever) [inline README.md:12]: Should be able to use `{{.AppName}}` instead of `family-portal`, right?

**Review by adamvictorclever — COMMENTED** [review]
Just one small thing that I think was missed!

File: README.md (hunk 1 above)
Comment (arsalansufi) [reply README.md]: 🤦
Thanks for the catch!

**Review by adamvictorclever — APPROVED** [review]
LGTM! 😄

**Pull request merged**